Example/FinancialConnections Example/Project.swift

```diff
@@ -29,7 +29,7 @@ let project = Project(
             productName: "FinancialConnectionsExample",
             bundleId: "com.stripe.example.Connections-Example",
             infoPlist: "FinancialConnections Example/Info.plist",
-            sources: "FinancialConnections Example/*.swift",
+            sources: "FinancialConnections Example/**/*.swift",
```

Comment:
This fixes a bug. During Tuist migration, the `Playground` folder was not included which maybe makes the Playground not run properly.